deploy/helm/sumologic/values.yaml

```diff
@@ -164,6 +164,12 @@ sumologic:
   fluentd:
     ## Option to specify the Fluentd buffer as file/memory.
     buffer: "memory"
+    ## Option to turn autoscaling on for fluentd and specify metrics for HPA.
+    autoscaling:
+      enabled: true
+      minReplicas: 1
```

Comment:
do we want to set `minReplicas` to `3` to match our current 3 replicas?

Reply:
Is was wondering if that's necessary? I was thinking that we keep the min to be 1 and let the autoscaler do its job.

Reply:
Yeah, it's worth a discussion. Should get @frankreno 's opinion as well.  Would 1 replica no longer be considered high availability (HA) since there could be data loss until the next replica is brought up?

Reply:
I would suggest we have a min of 2 or 3 at least to prevent any loss of data while its spinning up or if it did scale down to 1 and the node died, it would have a fall back.  This can be our default and we just expose it they values.yaml so customers can change if needed.

Reply:
ok will change it to 3 then.